test/helpers.bash

```diff
@@ -606,16 +607,16 @@ function check_conmon_cpuset() {
         fi
 
         found_cpuset=$(cat "$cpuset_path/pod_123-456/crio-conmon-$ctr_id/cpuset.cpus")
-        if [ -z "$cpuset" ]; then
+        if [ -z "${cpuset:-}" ]; then
```

Comment:
Not needed (we ensure cpuset is defined above)

Reply:
Fixed.